content/Services/RHOBS/rules-and-alerting.md

```diff
@@ -0,0 +1,157 @@
+# Rules and alerting capabilities
+
+## Overview
+
+As explained in more details [here](README.md), RHOBS features a deployment of [Observatorium](../../Projects/Observability/observatorium.md).
+
+Through the Observatorium API, tenants are able to **write** and **read** their own Prometheus [recording](https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/) and [alerting](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/) rules via the [Observatorium Rules API](https://github.com/observatorium/observatorium/tree/main/docs/design/rules-api.md).
+
+In addition to this, each of the RHOBS instances has an [Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/) deployed, which makes possible for tenants to configure custom alert routing configuration to route firing alerts to their specified receivers.
+
+## Goal
+
+This page aims to provide a simple tutorial of how a tenant can create an alerting rule via the Observatorium Rules API and configure Alertmanager properly to get alerted via a desired receiver.
+
+For this tutorial we will be using the `rhobs` tenant in the **stage environment**. URLs may change slightly in case another tenant is used.
+
+### Run and configure token-refresher
+
+To have access to the [Observatorium API](https://github.com/observatorium/api), the tenant making the requests needs to be correctly authenticated. For this you will need to run [token-refresher](https://github.com/observatorium/token-refresher) - a helper that fetches and refreshes OAuth2 access tokens via OIDC.
+
+First, clone the repo:
+
+```bash
+git clone git@github.com:observatorium/token-refresher.git
+```
+
+Now inside the same folder you've cloned the repo, run an instance of token-refresher:
```

Comment:
If our guidance is to run a token refresher docker instance, we will not need to clone the repo above.